src/DBFTPlugin/DBFTPlugin.cs

```diff
@@ -44,55 +44,55 @@ public override void Dispose()
 
         protected override void Configure()
         {
-            settings ??= new Settings(GetConfiguration());
+            _settings ??= new Settings(GetConfiguration());
```

Comment:
should remove the class. so it would be
```csharp
_settings ??= new(GetConfiguration());
```

Reply:
This does not matter. Many places keep it, many remove it.